shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley.hs

```diff
@@ -27,8 +28,18 @@ instance CryptoClass.Crypto c => Era (ShelleyEra c) where
 
 type instance Value (ShelleyEra c) = Coin
 
+type TxBodyConstraints era =
+  ( NoThunks (TxBody era),
```

Comment:
should this name `TxBodyConstraints` have Shelley-Based in it somewhere, like `SBTxBodyConstraints`?
Or would you say it is sufficient to have it in the Shelley file and inside the ShelleyBased bundle?

Reply:
Yep, I tend to hew closely to the style which says:

- Module names are relevant. If you have something in a module called `Data.List`, there's no need to call it `listLength` - just `length` is fine. 
- Prefer qualified imports in many cases. You'll see that almost everywhere that `TxBodyConstraints` is used, it's done so qualified as `Shelley.TxBodyConstraints`